src/screens/level.py

```diff
@@ -342,6 +351,10 @@ def reset(self):
                 fruit.kill()
             if tree.alive:
                 tree.create_fruit()
+        for bush in self.bush_sprites:
+            for fruit in bush.fruit_sprites:
+                fruit.kill()
+                bush.create_fruit()
```

Comment:
Why do you destroy a fruit just to respawn it afterwards? I'm a bit confused here...

Reply:
I basically just copied what was done for the trees. I assume it's to reset the number of fruit each new day, to make it a bit more random. And if the player had left one berry on each bush, that would also replenish